SpatialGDK/Source/SpatialGDK/Private/Interop/CrossServerRPCSender.cpp

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/CrossServerRPCSender.h"
+
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Utils/SpatialMetrics.h"
+
+using namespace SpatialGDK;
+
+CrossServerRPCSender::CrossServerRPCSender(ViewCoordinator& InCoordinator, USpatialMetrics* InSpatialMetrics)
+	: Coordinator(InCoordinator)
+	, SpatialMetrics(InSpatialMetrics)
+{
+}
+
+void CrossServerRPCSender::SendCommand(const FUnrealObjectRef& InTargetObjectRef, UObject* TargetObject, UFunction* Function,
+									   RPCPayload&& InPayload, FRPCInfo Info, const TOptional<Trace_SpanId>& SpanId) const
+{
+	if (Function == nullptr || TargetObject == nullptr || InTargetObjectRef.Entity == SpatialConstants::INVALID_ENTITY_ID
+		|| Info.Type != ERPCType::CrossServer)
+	{
+		return;
```

Comment:
Some of these errors seem more plausible than others, do we want to be warning/erroring in certain instances?